test/test-rosidl-message-generator.js

```diff
@@ -28,20 +28,9 @@ describe('ROSIDL Node.js message generator test suite', function(){
     rclnodejs.shutdown();
   });
 
-  it('Try require all message classes', function() {
-    this.timeout(10 * 1000);
-    const msgTypeList = rclnodejs.getAllMessageTypes();
-    const DEFAULT_NUMBER_OF_MESSAGES = 127; // # of a new standard ROS 2.0 build
-    assert(msgTypeList.length >= DEFAULT_NUMBER_OF_MESSAGES);
-    msgTypeList.forEach((msgType) => {
-      const MessageClass = message.getMessageClass(msgType);
-    });
-  });
-
-  it('Try use stdmsgs/msg/String.msg, using getMessageClass() function overrides', function() {
-    const MessageClass1 = message.getMessageClass(message.getMessageType('std_msgs', 'msg', 'String'));
-    assert.equal(MessageClass1.name, 'std_msgs__msg__String');
-    let msg = new MessageClass1();
+  it('Try use std_msgs/msg/String.msg', function() {
```

Comment:
I think we need to put a complete list here, and try require every message class in the list, to make sure at least the message class generation is OK